packages/apps/shopify-app-remix/README.md

```diff
@@ -218,6 +218,51 @@ const shopify = shopifyApp({
 - [Using Shopify managed installation](https://shopify.dev/docs/apps/auth/installation#shopify-managed-installation)
 - [Configuring access scopes through the Shopify CLI](https://shopify.dev/docs/apps/tools/cli/configuration)
 
+### Testing your app
+
+This package exports a helper method through `@shopify/shopify-app-remix/test-helpers` to simplify testing: `testConfig()`. This method can be used to pass dummy configuration properties to `shopifyApp()`.
+
+If your testing framework supports setting environment variables, we recommend using an environment variable, for example "SHOPIFY_TESTING" to replace your default config with the config returned from `testConfig()`.
+
+```ts
+// my-app/app/shopify.server.ts
+import { testConfig } from "@shopify/shopify-app-remix/test-helpers";
+...
+const config = {
+  ...
+};
+
+if (process.env.SHOPIFY_TESTING) {
+  Object.assign(config, testConfig());
```

Comment:
I don't follow. Would you mind clarifying?

Reply:
These are not currently being returned from `testConfig()`:
- adminApiAccessToken
- privateAppStorefrontAccessToken

What values would you like to return for these?
 
My understanding is that in addition to:
- apiKey
- apiSecretKey
- appUrl
- sessionStorage

we would also need to return `scopes` because it requires an environmental variable that's set by the Shopify CLI. Is that correct?